.github/workflows/ci.yaml

```diff
@@ -4,13 +4,15 @@ on:
   pull_request:
   push:
 
-concurrency:
-  group: ${{ github.ref_name }}
-  cancel-in-progress: true
-
 jobs:
   test:
     name: Tests
-    uses: theforeman/actions/.github/workflows/test-gem.yml@v0
-    with:
-      test: 'bundle exec rake test'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Tests
+        uses: theforeman/actions/.github/workflows/test-gem.yml@v0
+
+      - name: Run tests
+        run: |
+          bundle exec rake test
```

Comment:
@ekohl can you help me to understand this, what is wrong here if I'm doing it this or like what would be correct way to do it? I tried multiple ways but it kept failing.

Reply:
https://docs.github.com/en/actions/using-workflows/reusing-workflows#passing-inputs-and-secrets-to-a-reusable-workflow documents how passing inputs to reusable workflows works, but I've also just added an example: https://github.com/theforeman/actions#gem-test

Reply:
is it checking for ruby 3.2 too? do we want to check this?

Reply:
i was looking at this one https://github.com/archanaserver/kafo_parsers/actions/runs/5611152228
but seems like i need update development dependencies versions on gemspec to pass the test.

Reply:
@ekohl does this changes https://github.com/theforeman/kafo_parsers/commit/a1a41b07d633a21ae580b772f357636ecad183e6 looks good? I guess this is all steps we need right?